### PR TITLE
- Remove unused members.

### PR DESCRIPTION
**Pull request by baldersheim**
- Use the same ServiceLayerComponent.
- GC unused code.

@vekterli or @geirst PR

### CODE REVIEW

**Review by vekterli — APPROVED** [review]
:+1:

File: storage/src/vespa/storage/persistence/persistencethread.cpp [hunk 1]
```diff
@@ -92,6 +92,11 @@ class ResultTaskOperationDone : public spi::OperationComplete {
     vespalib::ISequencedTaskExecutor::ExecutorId   _executorId;
 };
 
+vespalib::string
+createThreadName(size_t stripeId) {
+    return vespalib::make_string("Thread %zu", stripeId);
```
Comment (geirst) [inline storage/src/vespa/storage/persistence/persistencethread.cpp:97]: Consider naming it "Persistence Thread x"

**Review by geirst — COMMENTED** [review]
👍

File: storage/src/vespa/storage/persistence/persistencethread.cpp (hunk 1 above)
Comment (baldersheim) [reply storage/src/vespa/storage/persistence/persistencethread.cpp:97]: Fixed in https://github.com/vespa-engine/vespa/pull/14920

**Pull request merged**